<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTask</class>
 <widget class="QDialog" name="AddTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="data.qrc">
    <normaloff>:/img/SchadierCorpIcon.PNG</normaloff>:/img/SchadierCorpIcon.PNG</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="lblEnd">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Ende</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QDateTimeEdit" name="dateEEnd">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="lblDuration">
      <property name="text">
       <string>Dauer</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QSpinBox" name="sbDuration">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>8766</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Strunden</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>125</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="lblStatus">
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comBState">
      <item>
       <property name="text">
        <string>TODO</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PROGRESS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DONE</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="lblStart">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Begin</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QDateTimeEdit" name="dateEStart">
      <property name="currentSection">
       <enum>QDateTimeEdit::HourSection</enum>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="lblTitle">
      <property name="text">
       <string>Titel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leTitle">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>391</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Aufgabe Hinzuf√ºgen oder Bearbeiten</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancle">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Abbrechen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnApply">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Speichern</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>leTitle</tabstop>
  <tabstop>comBState</tabstop>
  <tabstop>textEdit</tabstop>
 </tabstops>
 <resources>
  <include location="data.qrc"/>
 </resources>
 <connections/>
</ui>
